canister/src/heartbeat.rs

```diff
@@ -17,6 +17,10 @@ use ic_btc_types::{Block, BlockHash};
 /// The heartbeat fetches new blocks from the bitcoin network and inserts them into the state.
 pub async fn heartbeat() {
     print("Starting heartbeat...");
+
+    cycles_burn();
+    print("Done burning received cycles.");
```

Comment:
I'd remove this print statement, as it can quite noisy and burning cycles isn't very important for us from a logging perspective - we can already monitor it in metrics.